src/Handling/SpellCheckMiddleware.php

```diff
@@ -28,20 +30,21 @@ public function process(HTTPRequest $request, callable $delegate)
         HTMLEditorConfig::get($editor)->enablePlugins('spellchecker');
         HTMLEditorConfig::get($editor)->addButtonsToLine(2, 'spellchecker');
         $token = SecurityToken::inst();
-        HTMLEditorConfig::get($editor)->setOption('spellchecker_rpc_url', $token->addToUrl('spellcheck/'));
-        HTMLEditorConfig::get($editor)->setOption('browser_spellcheck', false);
-        HTMLEditorConfig::get($editor)->setOption(
-            'spellchecker_languages',
-            '+'.implode(', ', $this->getLanguages())
-        );
+        HTMLEditorConfig::get($editor)
+            ->setOption('spellchecker_rpc_url', Director::absoluteURL($token->addToUrl('spellcheck/')))
```

Comment:
Is absolute url necessary? Some subsites-enabled sites might have multiple domains so we try to use relative urls if possible.